supervisor/hassos.py

```diff
@@ -55,15 +55,23 @@ def board(self) -> Optional[str]:
         """Return board name."""
         return self._board
 
-    async def _download_raucb(self, version: AwesomeVersion) -> Path:
-        """Download rauc bundle (OTA) from github."""
+    def _get_download_url(self, version: AwesomeVersion) -> str:
         raw_url = self.sys_updater.ota_url
         if raw_url is None:
             raise HassOSUpdateError("Don't have an URL for OTA updates!", _LOGGER.error)
-        url = raw_url.format(version=str(version), board=self.board)
 
+        update_board = self.board
+
+        # OS version 6 and later renamed intel-nuc to generic-x86-64...
+        if update_board == "intel-nuc" and version.section(0) >= 6:
```

Comment:
```suggestion
        if update_board == "intel-nuc" and version >= 6:
```

Reply:
That seems not to work with the current rc releases:
```
>>> test = awesomeversion.AwesomeVersion("6.0.rc1")
>>> test > 6
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.9/site-packages/awesomeversion/awesomeversion.py", line 117, in __gt__
    raise AwesomeVersionCompare(
awesomeversion.exceptions.AwesomeVersionCompare: Can't compare unknown
```

Reply:
I fixed that a few weeks ago, make sure your installed requirements matches the requirements file

Reply:
This is with `21.5.0` which is the version pinned for the supervisor
```
>>> test = AwesomeVersion("6.0.rc1")
>>> test > 6
True
>>> test >= 6
True
```